.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node_js
+
+script:
+	- npm run lint
```

Comment:
Evidently [tabs aren't valid in YML](https://yaml.org/faq.html). Can we coerce our tooling to handle this distinction?

Reply:
ohhh interesting -- I'll update editorconfig to reflect that.